include/uapi/linux/mshv.h

```diff
@@ -286,6 +286,16 @@ struct mshv_invlpgb {
 	__u32 ecx;
 } __packed;
 
+
+struct mshv_kick_cpus {
+	__u64 len;
+	__u64 cpu_mask_ptr;	/* pointer to cpu mask bits */
+	__u64 flags;
+} __packed;
+
+#define MSHV_KICK_CPUS_FLAG_WAIT_FOR_CPUS	(1 << 0)
+#define MSHV_KICK_CPUS_FLAG_CANCEL_CPU_RUN  (1 << 1)
```

Comment:
Inconsistent formatting here and elsewhere in this file.

Reply:
hmm probably editor tabs issue, fixing...